modules/local/PEPPAN/main.nf

```diff
@@ -0,0 +1,48 @@
+process PEPPAN {
+    tag "$meta.id"
+    label 'process_medium'
+    conda '$moduleDir/environment.yml'
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://cidgoh/peppan:latest' :
+        'cidgoh/peppan:latest' }"
+
+    input:
+    tuple val(meta), path(gff)
+    path (reference_gff)
+
+    output:
+    tuple val(meta), path("*.gff"), path("*.fna")   ,             emit: gff_peppan
+    path "versions.yml"                             ,             emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def gff = params.reference_gff ? "-P params.reference_gff" : ''
+    def version = "1.0.5" // ------> update manually
```

Comment:
this should not be done manually. Given that the peppan is installed in the docker we can find out the version.

See example - https://github.com/cidgoh/nf-seqqc/blob/4b83e5e05c042e0352167f3ef915f8ce444aadb3/modules/nf-core/fastp/main.nf#L99